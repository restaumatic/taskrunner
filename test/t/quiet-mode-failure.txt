# quiet
echo "This output should be shown because the command fails"
echo "Second line of output"
exit 1