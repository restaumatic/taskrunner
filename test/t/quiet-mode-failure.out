-- output:
[toplevel] stdout | This output should be shown because the command fails
[toplevel] stdout | Second line of output
-- exit code: 1
